### ISSUE TITLE
Not able to see account theme to select on keycloak 22

### ISSUE THREAD

**Issue by navneetlal** [keycloakify/keycloakify#385]
```bash
#!/usr/bin/env bash

docker rm keycloak-testing-container || true

cd "/home/xxxxxxxxxx/build_keycloak"

docker run \
   -p 8080:8080 \
   --name keycloak-testing-container \
   -e KEYCLOAK_ADMIN=admin \
   -e KEYCLOAK_ADMIN_PASSWORD=admin \
   -e JAVA_OPTS=-Dkeycloak.profile=preview \
   -v "/home/xxxxxxxxxx/build_keycloak/src/main/resources/theme/keycloakify-starter":"/opt/keycloak/themes/keycloakify-starter":rw \
   -v "/home/xxxxxxxxxx/build_keycloak/src/main/resources/theme/keycloakify-starter-variant-1":"/opt/keycloak/themes/keycloakify-starter-variant-1":rw \
   -it quay.io/keycloak/keycloak:22.0.0 \
   start-dev

```

![image](https://github.com/keycloakify/keycloakify/assets/37542758/abf4fd5d-4b72-4d2d-90fe-a6fda00186f9)

---

but this works if i change the keycloak version to 21

**Comment by garronej** [comment]
Hi @navneetlal,  

Can you try with 22.0.1 ?  

I just tied and everything seems to be working as expected.  

<img width="1092" alt="image" src="https://github.com/keycloakify/keycloakify/assets/6702424/0cf194cf-adfb-4bce-80e6-1392a845e7af">

![image](https://github.com/keycloakify/keycloakify/assets/6702424/fd55659d-19d5-43dd-b4d5-d4d0c90a4e1c)


<img width="1181" alt="image" src="https://github.com/keycloakify/keycloakify/assets/6702424/973e3255-7ea7-4a33-8880-0995267c7601">

**Comment by navneetlal** [comment]
@garronej Hey the login theme is working. able to see option in dropdown. but in account theme i see no option to select my new theme.

![image](https://github.com/keycloakify/keycloakify/assets/37542758/1f3f772d-1cd9-4413-9abd-f49e537d9fc4)
![image](https://github.com/keycloakify/keycloakify/assets/37542758/4c88c0e4-e04b-4af4-8ff5-73504998306d)

Tried with 22.0.1 also

**Comment by garronej** [comment]
Sorry for missreading your issue and thank you for reporting.  
It's fixed in 7.15.4 just being released.

**Comment by garronej** [comment]
My sincere apologies,

Regrettably, I'm unable to support the Account theme in Keycloak 22. The original Account theme v1 has been phased out by Keycloak, and the current methods of implementing account and login themes have diverged significantly. 

I've reached out to the Keycloak development team in an effort to identify a possible resolution. However, I must candidly admit that a prompt solution might not be immediately forthcoming.

Here are the alternative courses of action you may wish to consider:

1. Develop a custom account theme in line with Keycloak's established method. The language employed is TypeScript/React which could be a silver lining for some. However, you would need to navigate the building and bundling process independently. Please refer to the following resource for more information: [Keycloak's Official Repository](https://github.com/keycloak/keycloak/tree/main/themes/src/main/resources/theme/keycloak.v2/account).

2. Continue using Keycloak 21 while I work on devising a viable solution.

In hindsight, I regret not having foreseen this situation and deeply apologize for any inconvenience this may cause. I appreciate your understanding and patience in this matter.
